CMake/Dependencies/libsrtp-CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ ExternalProject_Add(project_libsrtp
     GIT_REPOSITORY    https://github.com/cisco/libsrtp.git
     GIT_TAG           bd0f27ec0e299ad101a396dde3f7c90d48efc8fc
     PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/build
+    GIT_PROGRESS      TRUE
```

Comment:
Note for later:

Can't add GIT_SHALLOW to these since it doesn't reference a branch or tag (specific commit hash). We can use it when upgrading to later.